package.json

```diff
@@ -43,15 +43,15 @@
     "eth-testing": "^1.0.0",
     "husky": "^7.0.4",
     "jest": "^28.1.0",
-    "lerna": "^4.0.0",
+    "lerna": "^5.4.3",
     "lint-staged": "^12.2.2",
     "node-ts": "^5.1.2",
     "prettier": "^2.5.1",
     "prompt": "^1.2.1",
     "semver": "^7.3.5",
     "ts-jest": "^28.0.2",
     "tsc-watch": "^4.6.0",
-    "typescript": "^4.5.5"
+    "typescript": "4.7.4"
```

Comment:
This doesnt seem like a good idea to me. why force the exact version?

Reply:
Changed it back but I would probably prefer a tilda. I personally have had issues with carets causing unpredictability.